test/e2e/serial/tests/workload_placement_resources.go

```diff
@@ -0,0 +1,213 @@
+		Expect(err).ToNot(HaveOccurred())
+
+		// Note that this test, being part of "serial", expects NO OTHER POD being scheduled
+		// in between, so we consider this information current and valid when the It()s run.
+	})
+
+	AfterEach(func() {
+		err := e2efixture.Teardown(fxt)
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	// note we hardcode the values we need here and when we pad node.
+	// This is ugly, but automatically computing the values is not straightforward
+	// and will we want to start lean and mean.
+
+	Context("with at least two nodes suitable", func() {
+		// FIXME: this is a slight abuse of DescribeTable, but we need to run
+		// the same code which a different test_id per tmscope
+		DescribeTable("[tier1][ressched] a guaranteed pod with one container should be placed and aligned on the node",
+			func(tmPolicy nrtv1alpha1.TopologyManagerPolicy, requiredRes, expectedFreeRes corev1.ResourceList) {
+				nrtCandidates, targetNodeName := setupNodes(fxt, desiredNodesState{
+					NRTList:           nrtList,
+					RequiredNodes:     2,
+					RequiredNUMAZones: 2,
+					RequiredResources: requiredRes,
+					FreeResources:     expectedFreeRes,
+				})
+
+				nrts := e2enrt.FilterTopologyManagerPolicy(nrtCandidates, tmPolicy)
+				if len(nrts) != len(nrtCandidates) {
+					Skip(fmt.Sprintf("not enough nodes with policy %q - found %d", string(tmPolicy), len(nrts)))
+				}
+
+				By("Scheduling the testing pod")
+				pod := objects.NewTestPodPause(fxt.Namespace.Name, "testpod")
+				pod.Spec.SchedulerName = serialconfig.Config.SchedulerName
+				pod.Spec.Containers[0].Resources.Limits = requiredRes
+
+				err := fxt.Client.Create(context.TODO(), pod)
+				Expect(err).NotTo(HaveOccurred(), "unable to create pod %q", pod.Name)
+
+				By("waiting for node to be up and running")
+				podRunningTimeout := 1 * time.Minute
+				updatedPod, err := e2ewait.ForPodPhase(fxt.Client, pod.Namespace, pod.Name, corev1.PodRunning, podRunningTimeout)
+				if err != nil {
+					_ = objects.LogEventsForPod(fxt.K8sClient, updatedPod.Namespace, updatedPod.Name)
+				}
+				Expect(err).NotTo(HaveOccurred(), "Pod %q not up&running after %v", pod.Name, podRunningTimeout)
+
+				By("checking the pod has been scheduled in the proper node")
+				Expect(updatedPod.Spec.NodeName).To(Equal(targetNodeName))
+
+				By(fmt.Sprintf("checking the pod was scheduled with the topology aware scheduler %q", serialconfig.Config.SchedulerName))
+				schedOK, err := nrosched.CheckPODWasScheduledWith(fxt.K8sClient, updatedPod.Namespace, updatedPod.Name, serialconfig.Config.SchedulerName)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(schedOK).To(BeTrue(), "pod %s/%s not scheduled with expected scheduler %s", updatedPod.Namespace, updatedPod.Name, serialconfig.Config.SchedulerName)
+			},
+			Entry("[tmscope:pod] with topology-manager-scope: pod, using memory as deciding factor",
+				nrtv1alpha1.SingleNUMANodePodLevel,
+				corev1.ResourceList{
```

Comment:
We know these numbers are true because we know the hw that the tests are about to running on?

Reply:
we know the nodes can provide these many resources because we check for that explicitely (https://github.com/openshift-kni/numaresources-operator/pull/278/files/43241e84fd511b57248ccc8ca44e422a8749e81f#diff-647d0ef14accefb26fabf52caefe63073e8d43b6b7cdb5b57856c2fb91db16f3R154 )

the value were determined with a bit of try/error. We need something that fits on a "typical" server (expecting 32+32 cores and ~64 GiBs RAM) and "big enough" to have the base load (~2 cores, ~5 GiB) to be  fraction of them.

Yes, there is a big amount of heuristic and try/error. Auto-tuning these values is (still) way harder, though.

Reply:
> we know the nodes can provide these many resources because we check for that explicitely (https://github.com/openshift-kni/numaresources-operator/pull/278/files/43241e84fd511b57248ccc8ca44e422a8749e81f#diff-647d0ef14accefb26fabf52caefe63073e8d43b6b7cdb5b57856c2fb91db16f3R154 )

Link is broken

Reply:
line 154 below